test/runtests.jl

```diff
@@ -1,10 +1,18 @@
 using BenchmarkProfiles
 using Plots
+using DataFrames
 using Test
 
 unicodeplots()
 T = 10 * rand(25, 3)
 performance_profile(T, ["a", "b", "c"], title="Test Profile", linestyles=[:solid, :dash, :dot])
+dfs  = DataFrame[]
+for col in eachcol(T)
+    push!(dfs,DataFrame(:perf_measure=>col))
+end
+cost(df) = df.perf_measure
+stats = Dict([:a, :b, :c].=>dfs)
```

Comment:
```suggestion
stats = Dict([:a, :b, :c] .=> dfs)
```
Also, cool, didn't know about `.=>`.

Reply:
Very cool indeed.